### PR TITLE
add wallet_prefill_addresses command

### PR DESCRIPTION
**Pull request by lyoshenka**
This is very useful when you're publishing multiple files at once (e.g. youtube sync)

### CODE REVIEW

**Comment by lyoshenka** [comment]
Not sure if I did https://github.com/lbryio/lbry/pull/960/files#diff-eafee006192cca2ffbca5ba91dc83eeeR1336 right

File: lbrynet/core/Wallet.py [hunk 1]
```diff
@@ -1321,6 +1321,23 @@ def get_address_balance(self, address, include_balance=False):
...
+    @defer.inlineCallbacks
+    def create_addresses_with_balance(self, num_addresses, amount, broadcast=True):
+        addresses = self.wallet.get_unused_addresses(account=None)
+        if len(addresses) > num_addresses:
+            addresses = addresses[:num_addresses]
+        elif len(addresses) < num_addresses:
+            for i in range(len(addresses), num_addresses):
+                address = self.wallet.create_new_address(account=None)
+                addresses.append(address)
+            yield self._save_wallet()
```
Comment (jackrobison) [inline lbrynet/core/Wallet.py]: Can you add a TODO to remove `_save_wallet` from lbrynet? (lbryum should take care of it)

File: lbrynet/core/Wallet.py [hunk 2]
```diff
@@ -1321,6 +1321,23 @@ def get_address_balance(self, address, include_balance=False):
...
+
+        outputs = [[address, amount] for address in addresses]
+        # should this be defer_to_thread or defer_succeed ??
+        tx = yield self._run_cmd_as_defer_to_thread('paytomany', outputs)
+        if broadcast and tx['complete']:
+            yield self._broadcast_transaction(tx)
```
Comment (jackrobison) [inline lbrynet/core/Wallet.py]: you should keep the result here:
```
if broadcast and tx['complete']:
    txid = yield self._broadcast_transaction(tx)
    tx['txid'] = txid
```

File: lbrynet/daemon/Daemon.py [hunk 3]
```diff
@@ -2376,6 +2376,33 @@ def jsonrpc_wallet_send(self, amount, address=None, claim_id=None):
...
+        """
+        Create new addresses, each containing `amount` credits
+
+        Usage:
+            wallet_prefill_addresses [--no_broadcast]
+                                     (<num_addresses> | --num_addresses=<num_addresses>)
+                                     (<amount> | --amount=<amount>)
+
+        Returns:
+            (bool) true if addresses were created successfully
```
Comment (jackrobison) [inline lbrynet/daemon/Daemon.py]: This returns a dict, not a bool.

**Comment by lyoshenka** [comment]
merged in beca640f896e32ce29bd97befb55f0bd774a149a

File: lbrynet/core/Wallet.py [hunk 4]
```diff
@@ -1321,6 +1321,23 @@ def get_address_balance(self, address, include_balance=False):
...
+            addresses = addresses[:num_addresses]
+        elif len(addresses) < num_addresses:
+            for i in range(len(addresses), num_addresses):
+                address = self.wallet.create_new_address(account=None)
+                addresses.append(address)
+            yield self._save_wallet()
+
+        outputs = [[address, amount] for address in addresses]
+        # should this be defer_to_thread or defer_succeed ??
+        tx = yield self._run_cmd_as_defer_to_thread('paytomany', outputs)
```
Comment (kaykurokawa) [inline lbrynet/core/Wallet.py]: this should be run with _run_cmd_as_defer_succeed

    # run commands as a defer.succeed,
    # lbryum commands should be run this way , unless if the command
    # only makes a lbrum server query, use _run_cmd_as_defer_to_thread()

File: lbrynet/daemon/Daemon.py (hunk 3 above)
Comment (kaykurokawa) [reply lbrynet/daemon/Daemon.py]: probablly should return a bool to be consistent with other wallet_ commands

**Pull request closed without merging**